views/tools.erb

```diff
@@ -1,5 +1,28 @@
 <h1 class="dcf-txt-h3" id="tools-list-table">Tools You Can Reserve</h1>
 
+<form class="dcf-form dcf-mb-6" id="find-controls">
+    <div class="dcf-grid dcf-col-gap-vw dcf-row-gap-6">
+        <div class="dcf-col-100% dcf-col-100%-end@md">
+            <div class="dcf-grid-full dcf-grid-halves@sm dcf-col-gap-vw">
+                <div>
+                    <label for="workshop-category">Workshop Category:</label>
+                    <select id="workshop-category" style="width:auto;margin-top:10px;" tabindex="1" name="workshop_category" class="dcf-input-select">
+                    <option value=""></option>
+                    <% Resource.category_options.each do |cat_id, name| %>
+                        <option <%= 'selected="selected"' if cat_id == workshop_category.to_i %> value="<%= cat_id %>"><%= name %></option>
+                    <% end %>
+                    </select>
+
+                    <%# Assign the selected category to the `workshop_category` variable. %>
+                    <input type="hidden" name="hidden-workshop-category" id="hidden-workshop-category" value="<%= workshop_category %>">
```

Comment:
You don't need this hidden field. Hidden fields are typically just used to get values that are passed from the .rb into javascript. So like if you wanted to use that workshop_category variable value inside of some javascript code you would use a hidden field.